### PR TITLE
Feature/color icon by theme

### PR DESCRIPTION
**Pull request by dkmullen**
Ticket: #36408
Colors the ix logo with the theme's primary color. This branch does pull in a new node module that pulls ix.svg inline in the DOM so it can be styled with CSS.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "angular-calendar": "0.23.2",
     "angular-draggable-droppable": "^2.0.0",
     "angular-resizable-element": "^2.0.0",
+    "angular-svg-icon": "^5.1.1",
```
Comment (erinix) [inline package.json]: Remove the ^ from this line, looking at angular-svg-icon it seems newer versions are meant for angular 6, the ^ could introduce a non-working version of this dependency.

Comment (erinix) [inline package.json]: Remove the ^ from this line, looking at angular-svg-icon it seems newer versions are meant for angular 6, the ^ could introduce a non-working version of this dependency.

**Comment by erinix** [comment]
rebuild

**Comment by dkmullen** [comment]
rebuild

**Comment by dkmullen** [comment]
rebuild

**Pull request closed without merging**